docs-ref-conceptual/upcoming-breaking-changes.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Upcoming Breaking Changes – Azure CLI | Microsoft Docs
+description: Learn about breaking changes coming to Azure CLI in the next breaking change release
+author: dbradish-microsoft
+ms.author: dbradish
+manager: mkluck
+ms.date: 08/22/2022
+ms.topic: article
+ms.service: azure-cli
+ms.tool: azure-cli
+ms.custom: devx-track-azurecli, seo-azure-cli
+keywords: azure cli updates, azure cli notes, azure cli versions, azure cli breaking changes
+---
+
+# Upcoming breaking changes in Azure PowerShell
+
+## Az.AKS
+
+### `az acs`
+
+- The `acs` command group has been deprecated and as of 2.41.0, `acs` will be removed. For more information, please refer to [this doc](https://azure.microsoft.com/updates/azure-container-service-will-retire-on-january-31-2020/)
+  
+## Az.AppConfiguration
+
+### `az appconfig kv import`
+
+- Validations will be added to JSON file imports to ensure that only valid JSON objects are imported. Other JSON primitives will be rejected. 
+
+### `az appconfig kv export`
+
+- Validations will be added to JSON file imports to ensure that only valid JSON objects are imported. Other JSON primitives will be rejected. 
+
+## Az.AppService
+
+### `az logicapp create`
+
+- Parameter breaking-change will happen to the parameter sets
+  - `--consumption-plan-location` and `--os-type` parameter will be deprecated. 
+
+#TODO: Need to make sure this is all the information about this? Are the suggested replacements? Will these be completely removed? 
+
+## Az.KeyVault
```

Comment:
May I ask what is the display order of those modules? Could we keep the alphabetical order as our history notes and Power Shell doc? [upcoming-breaking-changes](https://learn.microsoft.com/en-us/powershell/azure/upcoming-breaking-changes?view=azps-8.2.0)